### PR TITLE
Add PIP and Docker Install Instructions

### PR DESCRIPTION
**Pull request by dmlb2000**
This adds a document describing pip install instructions along with dependencies for a single system install. The other document describes how to install using docker compose.

### CODE REVIEW

**Comment by dmlb2000** [comment]
@makopov could you take a look at these instructions see if they will work.

File: development/PIP_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Python PIP Install Instructions
+
+Pacifica is highly recommended to use virtual environments to install manually.
```
Comment (makopov) [inline development/PIP_INSTALL.md]: This sentence structure doesn't quite read right.

Comment (makopov) [reply development/PIP_INSTALL.md]: It also says environments (plural), what environments and how many are needed

File: development/PIP_INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+Download and install ElasticSearch for your platform.
+
+For Redhat bases systems you should follow the Elasticsearch
+[install instructions](https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html).
+
+
+### Install Python
+
+Install Python for your platform most Linux systems have this done already.
```
Comment (makopov) [inline development/PIP_INSTALL.md:87]: Is this python 2 or 3?

File: development/PIP_INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+MetadataServer &
+PolicyServer &
+UniqueIDServer &
+ArchiveInterface &
+ProxyServer &
+IngestServer &
+CartServer &
+celery -A cart worker -l info &
+celery -A ingest.backend worker -l info &
+```
```
Comment (makopov) [inline development/PIP_INSTALL.md:156]: And then what? How does one verify that things are running?

**Review by makopov — CHANGES_REQUESTED** [review]
Just a few things I think should be addressed

File: development/PIP_INSTALL.md (hunk 1 above)
Comment (dmlb2000) [reply development/PIP_INSTALL.md]: Added some more specific docs on this, the answer is both, you can use many or you can use one.

File: development/PIP_INSTALL.md (hunk 2 above)
Comment (dmlb2000) [reply development/PIP_INSTALL.md:87]: I put some info on this, Python 2.7 is recommended for now, Python 3 is in the works.

File: development/PIP_INSTALL.md (hunk 3 above)
Comment (dmlb2000) [reply development/PIP_INSTALL.md:156]: Got a section on verification now, makes sense.

**Review by makopov — APPROVED** [review]

**Pull request merged**